api/src/main/java/au/org/aodn/nrmn/restapi/dto/observableitem/ObservableItemGetDto.java

```diff
@@ -37,11 +37,11 @@
     @Schema(title = "Superseded By")
     private String supersededBy;
 
-    @Schema(title = "Superseded By Names")
+    @Schema(title = "Superseded Names")
     private String supersededByNames;
 
-    @Schema(title = "Superseded By IDs")
-    private String supersededByIDs;
+    @Schema(title = "Superseded IDs")
+    private String supersededByIds;
```

Comment:
We should change the variable references as well - they're supersededIds and names not supersededByIds and NAmes

Reply:
My mistake by the way.

Reply:
Agree.